Source/WebKit/GPUProcess/GPUConnectionToWebProcess.h

```diff
@@ -160,6 +160,8 @@ class GPUConnectionToWebProcess
     PAL::SessionID sessionID() const { return m_sessionID; }
 
     bool isLockdownModeEnabled() const { return m_isLockdownModeEnabled; }
+    bool isLockdownSafeFontParserEnabled() const { return m_preferences.isLockdownSafeFontParserEnabled; }
```

Comment:
Do these need to be separate values?  Can they ever be different?

Reply:
Yes, they are independent. The SafeFontParser can be disabled even in during lockdown mode, since it's controlled by an experimental feature flag.